pkg/morph/client/constructor.go

```diff
@@ -145,8 +145,10 @@ func New(key *keys.PrivateKey, opts ...Option) (*Client, error) {
 }
 
 func (c *Client) newCli(endpoint string) (*rpcclient.WSClient, *actor.Actor, error) {
-	cli, err := rpcclient.NewWS(c.cfg.ctx, endpoint, rpcclient.Options{
-		DialTimeout: c.cfg.dialTimeout,
+	cli, err := rpcclient.NewWS(c.cfg.ctx, endpoint, rpcclient.WSOptions{
+		Options: rpcclient.Options{
+			DialTimeout: c.cfg.dialTimeout,
```

Comment:
now sure what do we do with `CloseNotificationChannelIfFull`

Reply:
It's a deadlock safeguard basically, likely should be set. @AnnaShaleva?

Reply:
> It's a deadlock safeguard basically

yes, but do we need it? it leads to unexpected connection switches, not sure if it is ok for the current logic. but if we want if, i will recheck it

Reply:
> it leads to unexpected connection switches...

...or a deadlock. But you can choose which one you like more!

Reply:
well, not sure what is worse fot debugginh: unexpected connection switch or unexpected deadlock so added a commit that turns that new option on